### PR TITLE
Adding sample template with the read me and screenshots

### PR DESCRIPTION
**Pull request by pablothx-vmware**
Adding a project sample starter template to incorporate in Visual Studio.

### CODE REVIEW

File: Project Template/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# VMware Project Template in Visual Studio for .NET Automation SDK
+## Table of Contents
```
Comment (vinpai) [inline Project Template/README.md]: We don't need such detailed instructions. Please keep it short to include only the required steps.

File: Project Template/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- [Abstract](#abstract)
+- [Supported vCenter Releases](#supported-vcenter-releases)
+- [Table of Contents](#table-of-contents)
+- [Quick Start Guide](#quick-start-guide)
+  - [Setting up build environment](#setting-up-build-environment)
+  - [Setting up a vSphere Test Environment](#setting-up-a-vsphere-test-environment)
+  - [Building the Samples](#building-the-samples)
+
+## Abstract
```
Comment (vinpai) [inline Project Template/README.md]: No need of abstract. We already have this in the main README.

File: Project Template/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+This document describes how to import a VMWare Automation Project Template to Visual Studio 
+which has been developed to work with .NET Framework 4.5.
+With this template you will be able to have a basic structure to add your custom code and interact with the vAPI.
+
+## Supported vCenter Releases:
```
Comment (vinpai) [inline Project Template/README.md]: Not needed. Already present in main README.

File: Project Template/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## Quick Start Guide
+This document will walk you through getting up and running with the import a VMware Project Template in Visual Studio for .NET Automation SDK.
+
+### Setting up build environment
+To import the VMware Project Template, install Microsoft Visual Studio 2015 or higher. 
+
+* Copy the zip file (VMWare-SDK-Automation-Sample-Template.zip) into the the folder C:\Users\<username>\Documents\Visual Studio 2015\Templates\ProjectTemplates
+
+### Setting up a Project Template Test Environment
```
Comment (vinpai) [inline Project Template/README.md]: Why do you need vSphere test environment for the template ? This is not required. Please keep the instructions minimal. We already have detailed instructions in the main README.

File: Project Template/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+Please refer to the notes in each sample for detailed compatibility information.
+
+## Quick Start Guide
+This document will walk you through getting up and running with the import a VMware Project Template in Visual Studio for .NET Automation SDK.
+
+### Setting up build environment
+To import the VMware Project Template, install Microsoft Visual Studio 2015 or higher. 
+
+* Copy the zip file (VMWare-SDK-Automation-Sample-Template.zip) into the the folder C:\Users\<username>\Documents\Visual Studio 2015\Templates\ProjectTemplates
```
Comment (vinpai) [inline Project Template/README.md]: There are other ways to do this than manually copying. Why can't we create a vsix installer ? https://docs.microsoft.com/en-us/visualstudio/extensibility/getting-started-with-the-vsix-project-template

File: Project Template/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+To run the Project Template, a vSphere test environment is required with the following minimum configuration
+* 1 vCenter Server
+
+### Building and running the Project Template 
+
+1. Open Visual Studio
+2.  In the menu select File --> New --> Project...
+3. Select the template as the image below shows
+![Build Solution](screenshots/newProject.jpg?raw=true)
+4. Then right  click on the console project on the solution explorer and on the debug options add the "Command line arguments"
```
Comment (vinpai) [inline Project Template/README.md]: Instructions for how to "Build solution" is not needed. The instructions should end at the import step. We are not asking users to use the template, rather this is just a tool to enable them to get started quickly. You simply need instructions for how to add a new sample using the project template.

**Comment by pablothx-vmware** [comment]
Did all the changed to a new pull request.

**Pull request closed without merging**